stresstests/test/event_hub_helper.js

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+const EventHubClient = require('@azure/event-hubs').EventHubClient;
+const EventPosition = require('@azure/event-hubs').EventPosition;
+const debug = require('debug')('stresstests:EventHubHelper');
+
+/**
+ * A thenable that can be can be resolved or rejected from outside and records
```

Comment:
>thenable [](http://example.com/codeflow?start=5&length=8)

Nice.  Is this a word?

Reply:
It's a word in the JavaScript world: https://masteringjs.io/tutorials/fundamentals/thenable